actions.yaml

```diff
@@ -0,0 +1,6 @@
+# Copyright 2024 Canonical Ltd.
```

Comment:
From [previous PR](https://github.com/canonical/tmate-ssh-server-operator/pull/2)
[@jdkandersson] Is this a duplicate from above